### ISSUE TITLE
Package has no install method: qwtplot3d

### ISSUE THREAD

**Issue by ChrisFearn** [gnuradio/gr-inspector#27]
I am unable to install gr-inspector via Pybombs on Ubuntu 18.04.1.

I receive the following error: Package has no install method: qwtplot3d 

I am unable to find how to install this... qwtplot3d?

Help please

**Comment by sbmueller** [comment]
The official website and documentation, also how to install, can be found here http://qwtplot3d.sourceforge.net/

Also the whole PyBombs output would be helpful.

**Comment by ChrisFearn** [comment]
Mr Mueller,

Thanks for getting back to me. I am new to Pybombs and all this... I tried looking at the site but couldn't find how to install.  Google gave me lots of different avenues but none worked


The whole output..

chris@fearn:~$ pybombs install gr-inspector
[INFO] Prefix Python version is: 2.7.15
[INFO] PyBOMBS Version 2.3.4a0
[INFO] Phase 1: Creating install tree and installing binary packages:
[ERROR] Package has no install method: qwtplot3d


I really want to use your package for my Uni project

**Comment by sbmueller** [comment]
According to https://pkgs.org/download/qwt Ubuntu 18 does not have a package for qwtplot3d-qt4 anymore.

TL;DR
You need to downgrade to an older OS (Ubuntu 14 or 16 should do) or manually install qwtplot3d (and presumably many other dependencies) by hand.

To give some context: gr-inspector is an over 2 year old project which is not actively developed anymore (I shifted professionally and am not related to that topics any more, unfortunately). Ubuntu has a principle of fixed software versions in it's repos, meaning you are only able to install specific packages and versions that are provided by the apt-get package manager. Somewhen between when gr-inspector was developed and now, the Ubuntu repo maintainers decided to abandon the qwtplot3d-qt4 package. So either you downgrade to an older OS (Ubuntu 14 or 16) or go down the road and install all dependencies by hand (Google is your friend).

**Comment by ChrisFearn** [comment]
Ah that explains why nothing I found currently works...
I will do as suggested.  Really appreciate you taking the time to reply with such detail.

Now to return to my project trying to figure out how to implement 8-FSK and maybe ALE in GNU radio! Thanks

**Comment by sbmueller** [comment]
Closing since not a gr-inspector issue
